### DISCUSSION TITLE
[已解决]REALITY在模板区里的命名规范是？

### DISCUSSION THREAD

**Discussion by chika0801** [XTLS/Xray-core#1706] [Ideas]
参考 https://github.com/XTLS/Xray-examples 模板区

例如 VLESS-TCP-XTLS-Vision

现在的模板命名是 VLESS-XTLS-uTLS-REALITY

如果按照习惯是 VLESS-TCP-XTLS-Vision-REALITY-uTLS ，又太长了。

不知道有什么建议，或者就按 VLESS-XTLS-uTLS-REALITY 命名模板文件夹吗？

**Comment by cross-hello** [comment]
VLESS-TCP-REALITY-Vision

**Comment by RPRX** [comment]
我写的是 VLESS-XTLS-uTLS-REALITY，因为 XTLS 已经指代了 TCP 和 Vision
